Database/DDL.sql

```diff
@@ -0,0 +1,232 @@
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Database: `nur_real_estate`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `account`
+--
+
+CREATE TABLE `account` (
+  `account_id` varchar(36) NOT NULL,
+  `first_name` varchar(20) NOT NULL,
+  `last_name` varchar(20) NOT NULL,
+  `password` varchar(10) NOT NULL,
+  `role` varchar(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `address`
+--
+
+CREATE TABLE `address` (
+  `address_id` varchar(36) NOT NULL,
+  `property_id` varchar(36) NOT NULL,
+  `house_no` varchar(50) NOT NULL,
+  `city` varchar(100) NOT NULL,
+  `sub_city` varchar(100) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `admin`
+--
+
+CREATE TABLE `admin` (
+  `admin_id` varchar(36) NOT NULL,
+  `contact_id` varchar(36) NOT NULL,
+  `agent_id` varchar(36) NOT NULL,
+  `account_id` varchar(36) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `agent`
+--
+
+CREATE TABLE `agent` (
+  `agent_id` varchar(36) NOT NULL,
+  `nationality` varchar(50) NOT NULL,
+  `languages` varchar(100) NOT NULL,
+  `experience_since` datetime NOT NULL,
+  `contact_id` varchar(36) NOT NULL,
+  `admin_id` varchar(36) NOT NULL,
+  `account_id` varchar(36) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `contact`
+--
+
+CREATE TABLE `contact` (
+  `contact_id` varchar(36) NOT NULL,
+  `phone_no` varchar(15) NOT NULL,
+  `email` varchar(50) NOT NULL,
+  `whatsapp_no` varchar(15) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `property`
+--
+
+CREATE TABLE `property` (
+  `property_id` varchar(36) NOT NULL,
+  `title` varchar(255) NOT NULL,
+  `description` varchar(255) NOT NULL,
+  `price` float NOT NULL,
+  `property_type` varchar(255) NOT NULL,
+  `property_size` int(11) NOT NULL,
+  `no_bedrooms` int(11) NOT NULL,
+  `no_bathrooms` int(11) NOT NULL,
+  `has_maidsRoom` tinyint(1) NOT NULL,
+  `no_parking` int(11) NOT NULL,
+  `user_id` varchar(36) NOT NULL,
+  `agent_id` varchar(36) NOT NULL,
+  `listed_date` datetime NOT NULL,
+  `status` varchar(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `user`
+--
+
+CREATE TABLE `user` (
+  `user_id` varchar(36) NOT NULL,
+  `property_id` varchar(36) NOT NULL,
```

Comment:
You don't need this as well.. user and property has one to many relationship.. and you have already defined that in property line number 111.

Reply:
Done